portfolio/target/maven-archiver/pom.properties

```diff
@@ -1,5 +0,0 @@
-#Generated by Maven
```

Comment:
Can you add the target directory to the .gitignore file? As I understand, these are all generated when running mvn build, so it'd be better not to include them in version control. Let me know if you have any questions on that.